lib/internal/module.js

```diff
@@ -130,3 +121,12 @@ function addBuiltinLibsToObject(object) {
     });
   });
 }
+
+module.exports = exports = {
```

Comment:
That's my preference but in a separate issue @sam-github expressed a preference for these to retain the `= exports`